parameters/src/testnet_v1/mod.rs

```diff
@@ -0,0 +1,138 @@
+// Copyright (C) 2019-2023 Aleo Systems Inc.
+// This file is part of the snarkVM library.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+pub mod genesis;
+pub use genesis::*;
+
+const REMOTE_URL: &str = "https://s3-us-west-1.amazonaws.com/testnet-v1.parameters";
+
+// BondPublic
+impl_remote!(BondPublicProver, REMOTE_URL, "resources/", "bond_public", "prover");
```

Comment:
how are these parameters generated?

Reply:
These parameters are generated with the scripts in `parameters/scripts/canary/credits.sh`

These parameters are deterministically generated based on the SRS, so everyone should derive the same keys.